ios/Classes/SwiftFacebookAppEventsPlugin.swift

```diff
@@ -132,4 +135,10 @@ public class SwiftFacebookAppEventsPlugin: NSObject, FlutterPlugin {
         Settings.isAutoLogAppEventsEnabled = enabled
         result(nil)
     }
+    
+    private func handleSetIsDebugEnabled(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        let enabled = call.arguments as! Bool
+        Settings.enableLoggingBehavior(.appEvents)
```

Comment:
The `enabled` flag isn't used. And I was trying to find the right SDK configuration API for that and only found this.

https://github.com/facebook/facebook-ios-sdk/blob/a68760b8b04d4daebf87ba40920cce5228b9ffa5/FBSDKCoreKit/FBSDKCoreKit/FBSDKSettings.h#L104

Is that what should be used, perhaps? Should be something other than just logging behaviour, no?